pkgs/development/compilers/ghc/head.nix

```diff
@@ -5,15 +5,15 @@
   # If enabled GHC will be build with the GPL-free but slower integer-simple
   # library instead of the faster but GPLed integer-gmp library.
 , enableIntegerSimple ? false, gmp
+, version ? "8.3.20170808"
```

Comment:
I could override `src` with `overrideDerivation` but if I remember right, I couldn't do the same for `version`.